client/components/Recipes/RecipeDetails.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Button from '../common/Button.jsx';
+import { getOneRecipe } from '../../actions/recipeActions/recipeActions';
+import { Icons } from '../common/Icons.jsx';
+import RecipeCardImage from './RecipeCard/RecipeCardImage.jsx';
+import RecipeCardFooter1 from './RecipeCard/RecipeCardFooter1.jsx';
```

Comment:
RecipeFooter1???Use more approipriate variable naming ....RecipeFooter will be fine